sdk/python/packages/flet-core/src/flet_core/alert_dialog.py

```diff
@@ -207,14 +210,8 @@ def actions_alignment(self) -> MainAxisAlignment:
 
     @actions_alignment.setter
     def actions_alignment(self, value: MainAxisAlignment):
-        self.__actions_alignment = value
-        if isinstance(value, MainAxisAlignment):
-            self._set_attr("actionsAlignment", value.value)
-        else:
-            self.__set_actions_alignment(value)
-
-    def __set_actions_alignment(self, value: MainAxisAlignmentString):
-        self._set_attr("actionsAlignment", value)
+        self.__actions_alignment = get_valid_enum(MainAxisAlignment, value, _MainAxisAlignmentDefault)
+        self._set_attr("actionsAlignment", self.__actions_alignment.value)
```

Comment:
@Vlod-github I'm thinking on the entire approach here...
At the moment the idea behind those `None` or `MainAxisAlignment.NONE` or etc. is that if the property is not set it's not being sent over the wire to the server (Fletd), i.e. something like `{"t": "alertdialog"}` is sent if no properties are set.

When we set something non-None in the constructor it goes to `self._set_attr()` and will be set if just `AlertDialog()` constructor is used, something like `{"t": "alertdialog", "actionsAlignment": "end"}`, thus increasing overall traffic by sending unnecessary data.

What if further improve the logic around enum props and will be calling `self._set_attr()` with None if semantically default enum value is specified, kind of:

```python
self._set_attr("actionsAlignment", self.__actions_alignment.value if self.__actions_alignment != _MainAxisAlignmentDefault else None)
```

Do you see the point? Does it make sense?